risc0/zkvm/Cargo.toml

```diff
@@ -95,6 +95,7 @@ serial_test = "3.0"
 tar = "0.4"
 tempfile = "3"
 test-log = { version = "0.2", default-features = false, features = ["trace"] }
+risc0-groth16 = { workspace = true, features = ["docker"] }
```

Comment:
I think we need to drop the `features = ` here, and then forward the `docker` feature flag from this crate to `risc0-groth16`